src/otx/recipe/detection/dfine_x.yaml

```diff
@@ -90,7 +89,11 @@ overrides:
         - class_path: torchvision.transforms.v2.ToDtype
           init_args:
             dtype: ${as_torch_dtype:torch.float32}
-            scale: true
+            scale: false
```

Comment:
doesn't it work?

Reply:
Nope, current OTXDataModule assume std: (1.0, 1.0, 1.0) without Normalisation aug, however, when scale is set, std is set to (255.0, 255.0, 255.0). This assumption breaks OV inference due to inconsistency in trained model and OV model. I had to set it to false and add a Normalisation aug to fix this issue